modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-oadp-rosa-sts_{context}"]
+= OADP on ROSA with STS
+
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. A ROSA deployment of OpenShift is configured specifically for AWS services. AWS STS is a global web service that provides short-term credentials for IAM or federated users. ROSA with STS is the recommended credential mode for ROSA clusters.
+
+[The following sentence was in the source. Is it OK? is it too vague?] Not all of the features and options provided by OADP are recommended or supported while using ROSA clusters with OADP.
+
+[IMPORTANT]
+====
+Disable Restic for OADP in ROSA environments because Restic is not supported in such environments. Instead use the native snapshots workflow to backup volumes.
+====
+
+.Prerequisites
+
+* A ROSA OpenShift Cluster with the required access and tokens.
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/backup_and_restore/application-backup-and-restore#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
+* An OpenShift secret that has been created from your AWS token file.
++
+[source,yaml]
+----
+cat <<EOF > ${SCRATCH}/credentials
+[default]
+role_arn = ${ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+oc -n openshift-adp create secret generic cloud-credentials \
+  --from-file=${SCRATCH}/credentials
+----
+
+.Procedure
+
+. Install the OADP Operator.
+. Create AWS cloud storage using your AWS credentials:
++
+[source,yaml]
+----
+cat << EOF | oc create -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: CloudStorage
+metadata:
+  name: ${CLUSTER_NAME}-oadp
+  namespace: openshift-adp
+spec:
+  creationSecret:
+    key: credentials
+    name: cloud-credentials
+  enableSharedConfig: true
+  name: ${CLUSTER_NAME}-oadp
+  provider: aws
+  region: $REGION
+EOF
+----
+. Deploy the Data Protection Application:
```

Comment:
Instead of 
Deploy the Data Protection Application

I would say
Create the DataProtectionApplication resource, used to configure the connection to the storage where the backups and volume snapshots will be stored.

( you don't deploy the DataProtectionApplication, just create the CR and use it to setup the velero configuration )

Reply:
Thanks, @birsanv!